tests/e2e/workloads/couchbase/test_couchbase_node_reboot.py

```diff
@@ -49,34 +53,56 @@ def cb_setup(self, couchbase_factory_fixture):
             )
         ]
     )
-    def test_run_couchbase_node_reboot(self, cb_setup, nodes, pod_name_of_node):
+    def test_run_couchbase_node_reboot(
+            self, cb_setup, nodes, pod_name_of_node):
         """
         Test couchbase workload with node reboot
         """
+        # Check worker node utilization (adm_top)
+        get_node_resource_utilization_from_adm_top(
+            node_type='worker', print_table=True
+        )
+        get_node_resource_utilization_from_adm_top(
+            node_type='master', print_table=True
+        )
+
         if pod_name_of_node == 'couchbase':
             node_list = self.cb.get_couchbase_nodes()
         elif pod_name_of_node == 'osd':
             node_list = get_osd_running_nodes()
         elif pod_name_of_node == 'master':
-            node_list = get_master_nodes()
+            master_node = get_typed_nodes(pod_name_of_node, num_of_nodes=1)
 
-        node_1 = get_node_objs(
-            node_list[random.randint(0, len(node_list) - 1)])
+        # Restart relevant node
+        if pod_name_of_node == 'master':
+            nodes.restart_nodes(master_node, wait=False)
+            waiting_time = 40
+            log.info(f"Waiting {waiting_time} seconds...")
+            time.sleep(waiting_time)
+        else:
+            node_1 = get_node_objs(node_list[random.randint(0, len(node_list) - 1)])
```

Comment:
is there any other var like `node_2`? if not, shouldn't we call the var `node`?

Reply:
No node_2,  will update